static/openapi/prerendering-spec.json

```diff
@@ -2690,59 +3739,55 @@
-                          },
-                          "created": {
-                            "type": "string",
-                            "format": "date-time",
-                            "description": "Date when the settings was created"
-                          },
-                          "lastUpdated": {
-                            "type": "string",
-                            "format": "date-time",
-                            "description": "Date when the settings was updated"
-                          }
+                      "type": "object",
+                      "properties": {
+                        "_key": {
+                          "type": "string",
+                          "description": "Settings unique identifier"
                         },
-                        "required": [
-                          "_key",
-                          "origin",
-                          "injectInlineCSS",
-                          "addRequestHeader",
-                          "blockRequestTypes",
-                          "urlBlockList",
-                          "created"
-                        ]
-                      }
+                        "origin": {
+                          "type": "string",
+                          "format": "uri",
+                          "description": "Origin base URL"
+                        },
+                        "injectInlineCSS": {
+                          "type": "boolean",
+                          "description": "Whether or not to inject CSS into the HTML"
+                        },
+                        "addRequestHeader": {
+                          "type": "boolean",
+                          "description": "Whether or not to add X-PhotonIQ-Prerender-Request header to each request made during page rendering"
+                        },
+                        "blockRequestTypes": {
+                          "type": "array",
+                          "items": { "type": "string" },
+                          "description": "List of resource types that should be blocked (e.g., fonts, stylesheets)"
+                        },
+                        "urlBlockList": {
+                          "type": "array",
+                          "items": { "type": "string" },
+                          "description": "List of URLs that should be blocked (e.g., https://googleads.g.doubleclick.net/pagead)"
+                        },
+                        "created": {
+                          "type": "string",
+                          "format": "date-time",
+                          "description": "Date when the settings was created"
+                        },
+                        "lastUpdated": {
+                          "type": "string",
+                          "format": "date-time",
+                          "description": "Date when the settings was updated"
```

Comment:
Same as above. "settings was" is not correct.

Reply:
As we discussed, we're going to keep the `was`.